src/modules/party/party.service.ts

```diff
@@ -0,0 +1,63 @@
+import { HttpService } from '@nestjs/axios';
+import { Inject, Injectable } from '@nestjs/common';
+import { ConfigType } from '@nestjs/config';
+import AcbsConfig from '@ukef/config/acbs.config';
+import { lastValueFrom } from 'rxjs';
+
+import { AcbsGetPartiesBySearchTextResponseElement } from './dto/acbs-get-parties-by-search-text-response-element.dto';
+import { GetPartiesBySearchTextResponseElement } from './dto/get-parties-by-search-text-response-element.dto';
+import { GetPartiesBySearchTextException } from './exception/get-parties-by-search-text.exception';
+
+@Injectable()
+export class PartyService {
+  private static readonly getPartiesBySearchTextPath = '/Party/Search';
+
+  constructor(
+    @Inject(AcbsConfig.KEY)
+    private readonly config: Pick<ConfigType<typeof AcbsConfig>, 'baseUrl'>,
+    private readonly httpService: HttpService,
+  ) {}
+
+  async getPartiesBySearchText(token: string, searchText: string): Promise<GetPartiesBySearchTextResponseElement[]> {
+    if (searchText === null) {
```

Comment:
I may have missed this in the other PR, but if `searchText` gets validated in the DTO, then do we still need this here?